### DISCUSSION TITLE
How can the partial factors be adapted in D-Settlement?

### DISCUSSION THREAD

**Discussion by wjvanos** [Deltares/GEOLib#169] [Q&A]
The main issue is that the partial factors for EC7 NL RC 1,2,3 for constructive loads are not correct in the output file (.shi).

In the Geolib packages these partial factors are set in geolib/models/dsheetpiling/internal_partial_factors.py. Is there some reason the parameters for constructive loads (_constructloadfactorpermfav, constructloadfactorvarfav, constructloadfactorpermunfavcalc_ and _constructloadfactorvarunfavcalc_) are set to 1.0 instead of the values from the Eurocode?

How can these partial factors be adapted?
I tried already the following things:
- Add the parameters via _set_calculation_options_. However, the class _CalculationOptions_ has no variable to adapt the partial factors.
- When I adapt the model (with _model.datastructure.input_data.calculation_options_), this does not end up in the .shi-file.

All suggestions are welcome!

**Comment by wfaustmann** [comment]
I presume the title should refer to D-Sheet Piling instead of D-Settlement
There was an issue related to this which was fixed in [pr 154](https://github.com/Deltares/GEOLib/issues/154). This is fixed in the latest Geolib release. Could be check which version you are using and if this issue is still relevant?
